Pipfile

```diff
@@ -25,8 +22,7 @@ uvicorn = "*"
 pydantic = "*"
 "ga4gh.vrs" = {version = ">=0.9.0a0", extras = ["extras"]}
 gene-normalizer = ">=0.2.9"
-pyliftover = "*"
 boto3 = "*"
 "ga4gh.vrsatile.pydantic" = ">=0.1.0.dev9, < 0.2.0"
-cool-seq-tool = ">=0.1.13"
+cool-seq-tool = ">=0.1.14.dev0"
```

Comment:
confirm: this is correct?

Reply:
Yes! We want to use the new cool-seq-tool updates